### DISCUSSION TITLE
SPX v1.15.3-rc2 - Testing & Call Outs

### DISCUSSION THREAD

**Discussion by Reiers** [filecoin-project/lotus#8717] [Show and tell]
![banner](https://user-images.githubusercontent.com/63351350/170097020-c19a4cc9-0845-44a4-bf49-002e9fdd6ebe.png)

### Hello SPX Fellows

Upgrade to: [v1.15.3-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.15.3-rc2) and fill out the Reply card - report any bugs, if any. 

❗You need to upgrade Go to v1.17.9 or higher. 

🚀 This post if you are participating in this testing round. 
If you aren't able to join for reasons, please reach out to @jennijuju or me on Slack!!!

Improvements & bug fixes see [here.](https://github.com/filecoin-project/lotus/releases/tag/v1.15.3-rc2)

### Reply card:
- [ ] If you have successfully sealed some new sectors on this tag.
- [ ] If you have successfully made some Storage Deals on this tag.
- [ ] If you have successfully made some Snap Deals on this tag.
- [ ] If you passed your first WindowPoSt on this tag.
- [ ] If the log is back to normal. 
- [ ] The resource manager has not caused any connection problems.

## **Reminders and Risk Mitigations**

Questions are welcomed in this discussion, the team will be monitoring this discussion and check-in regularly.

Issues can be reported in https://github.com/filecoin-project/lotus/issues. Issues should be submitted with:

- Daemon logs / set `RUST_LOG=debug` if possible. 
- go routine dumps `lotus pprof goroutines` and `lotus-miner pprof goroutines`
- the operation/step at which the problem occurred
- The stack trace or logs as relevant. As always, for large log files, you may upload it in this Google Disk folder and link it in the issue!

❗ _If you think the v1.15.3-rc2 branch is not working well, especially the resource manager, keep all the logs for describing the issue and disable the resource manager with `LOTUS_RCMGR=0`, if the problem still persists revert back to the stable v1.15.2 release._

**Comment by TippyFlitsUK** [comment]
### Reply card:
- [ ] If you have successfully sealed some new sectors on this tag.
- [ ] If you have successfully made some Storage Deals on this tag.
- [ ] If you have successfully made some Snap Deals on this tag.
- [ ] If you passed your first WindowPoSt on this tag.
- [ ] If the log is back to normal. 
- [ ] The resource manager has not caused any connection problems.

**Comment by namoo1** [comment]
## Meest NAM Reply card: 
 
- [ ] If you have successfully sealed some new sectors on this tag.
- [ ] If you have successfully made some Storage Deals on this tag.
- [ ] If you have successfully made some Snap Deals on this tag.
- [ ] If you passed your first WindowPoSt on this tag.
- [ ] If the log is back to normal.
- [ ] The resource manager has not caused any connection problems.

**Comment by SBudo** [comment]
Reply card:
 
- [x] If you have successfully sealed some new sectors on this tag.
- [x] If you have successfully made some Storage Deals on this tag.
- [x] If you have successfully made some Snap Deals on this tag.
- [x] If you passed your first WindowPoSt on this tag.
- [x] If the log is back to normal.
- [x] The resource manager has not caused any connection problems.

**Comment by TippyFlitsUK** [comment]
I'm running into an issue with `v1.15.3-rc2`.

I am attempting to do `lotus client retrieve` from my own miner but getting the following error:

```
lotus client retrieve --provider f08403 --maxPrice 0 --allow-local --car 'QmajCHiSsjcHtStETf3jyiaM1gwMWT5Z3tkCZfW9SGWPUD' /filecoin-storage/offline/EverGreen/baga6e~vl4y26ey__QmajCH~W9SGWPUD.car

2022-05-25T03:48:20.894+0100	WARN	rpc	go-jsonrpc@v0.1.5/client.go:544	unmarshaling failed	{"message": "{\"Err\":\"exhausted 5 attempts but failed to open stream, err: peer:12D3KooWBwUERBhJPtZ7hg5N3q1DesvJ67xx9RLdSaStBz9Y6Ny8: resource limit exceeded\",\"Root\":null,\"Piece\":null,\"Size\":0,\"MinPrice\":\"\\u003cnil\\u003e\",\"UnsealPrice\":\"\\u003cnil\\u003e\",\"PricePerByte\":\"\\u003cnil\\u003e\",\"PaymentInterval\":0,\"PaymentIntervalIncrease\":0,\"Miner\":\"f08403\",\"MinerPeer\":{\"Address\":\"f08403\",\"ID\":\"12D3KooWBwUERBhJPtZ7hg5N3q1DesvJ67xx9RLdSaStBz9Y6Ny8\",\"PieceCID\":null}}"}
ERROR: RPC client error: unmarshaling result: failed to parse big string: '"\u003cnil\u003e"'
```


I am downgrading to `v1.15.3-rc1` now to confirm. @namoo1, @SBudo. would you guys be able to test a self retrieval on RC2?

Many thanks!

UPDATE: 
After downgrading to  `v1.15.3-rc1` I am able to successfully self-retrieve again, This appears to be an issue with RC2.

**Reply by SBudo** [reply]
Hey @TippyFlitsUK (sorry for the late reply).
I've been doing self-retrievals (through evergreen) with rc2 without any issues.

I did create a IP reroute as my retrieval node was on the same network as my miner node, but that shouldn't have impact the software version

**Reply by TippyFlitsUK** [reply]
Many thanks @SBudo!!

**Comment by tmyuu** [comment]
```
$ go version
go version go1.18.1 linux/amd64
$ lotus-miner version
Daemon:  1.15.3-rc2+mainnet+git.ddbcf1fc6+api1.5.0
Local: lotus-miner version 1.15.3-rc2+mainnet+git.ddbcf1fc6
$ boostd -h
VERSION: 0.0.1+git.bc85712
```
- [x]  If you have successfully sealed some new sectors on this tag.
- [x]  If you have successfully made some Storage Deals on this tag.
- [x]  If you have successfully made some Snap Deals on this tag.
- [x]  If you passed your first WindowPoSt on this tag.
- [x]  If the log is back to normal.
- [x]  The resource manager has not caused any connection problems.

**Comment by neondragon** [comment]
### Reply card:
* [x]  If you have successfully sealed some new sectors on this tag.
* [x]  If you have successfully made some Storage Deals on this tag.
* [ ]  If you have successfully made some Snap Deals on this tag.
* [x]  If you passed your first WindowPoSt on this tag.
* [ ]  If the log is back to normal.
* [x]  The resource manager has not caused any connection problems.

**Comment by jamerduhgamer** [comment]
### Reply card:

lotus-miner version
Daemon:  1.15.3-rc2+mainnet+git.ddbcf1fc6.dirty+api1.5.0
Local: lotus-miner version 1.15.3-rc2+mainnet+git.ddbcf1fc6.dirty

* [x]  If you have successfully sealed some new sectors on this tag.
* [ ]  If you have successfully made some Storage Deals on this tag.
* [ ]  If you have successfully made some Snap Deals on this tag.
* [x]  If you passed your first WindowPoSt on this tag.
* [x]  If the log is back to normal.
* [x]  The resource manager has not caused any connection problems.
